lib/rails_dev_tweaks/granular_autoload/middleware.rb

```diff
@@ -24,8 +24,8 @@ def call(env)
 
       # No-op if this is the first request.  The initializers take care of that one.
       if self.class.processed_a_request? && reload_dependencies?
-        ActionDispatch::Reloader.cleanup!
-        ActionDispatch::Reloader.prepare!
+        Rails.application.reloader.reload!
+        Rails.application.reloader.prepare!
```

Comment:
I think it just `Rails.application.reloader.reload!`

Reply:
This fix works for me, including both `reload!` and `prepare!`. Of course, since this has not been merged yet so I had to manually change it in the local `middleware.rb` file. It is important to note that this *only works in Rails 5*.